dependency.lic

```diff
@@ -49,10 +49,51 @@ class ArgParser
   end
 
   def display_args(data)
-    data.each do |def_set|
-      respond ''
-      respond "  ;#{Script.current.name} " + def_set.map { |x| format_item(x) }.join(' ')
-      def_set.each { |x| respond "   #{(x[:display] || x[:name]).ljust(12)} #{x[:description]} #{x[:options] ? '[' + x[:options].join(', ') + ']' : ''}"; }
+    # This section displays help output for a script. It recognizes when a script call has been given
+    # with the wrong number or incorrect arguments, or when a script has been called via
+    # ;script-name help. It outputs an example script call, including arguments, with optional
+    # arguments enclosed in brackets. If an optional script_summary argument has been specified within
+    # the script, it also outputs a summary of the script. Further, if yaml settings have been
+    # outlined in base-help.yaml, it will output help information for the yaml settings used.
+    # Example output when all help settings have been specified:
+    #
+    # >;circlecheck help
+    #   SCRIPT SUMMARY: Checks your circle and skills to next circle. Optionally checks skills for a target circle.
+
```

Comment:
🧹 nitpick, would you be ok to add `#` on the empty lines 62 and 68? I think it'd help show that all three comment blocks are part of the same example.

Reply:
Another good cosmetic catch.